bundle.Dockerfile

```diff
@@ -1,26 +1,12 @@
-#
-# Copyright Red Hat
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
 FROM scratch
 
 # Core bundle labels.
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=registry-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=beta
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.28.0
+LABEL operators.operatorframework.io.bundle.channels.v1=alpha
```

Comment:
Wonder if it matters that this is changing to `alpha`? I assume it is related to the sdk moving from `1.28.0` to `1.36.0`

Reply:
No it should be `beta`, need to run `export CHANNELS=beta` before running `make bundle`.

I think it would be worth making this a persistent setting so someone cutting the release doesn't have to set this every time it is not an alpha release. Added a comment to this step in our release process document as well.